app/assets/javascripts/modal/modal-workflow.js

```diff
@@ -26,6 +26,8 @@ ModalWorkflow.prototype.render = function (response) {
 
 ModalWorkflow.prototype.renderSuccess = function (result) {
   this.$multiSectionViewer.showDynamicSection(result.body)
+  var headingText = this.getSectionHeadingText()
```

Comment:
I think it would be better to avoid creating a temporary variable here - the function name should be descriptive enough.

```js
  this.setModalAriaLabel(this.getSectionHeadingText())
```

Reply:
I rely too much on garbage collector 😄 . I agree is clear enough if we use them as suggested. Updated.